submission/launch.sh

```diff
@@ -4,16 +4,15 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 trap "vagrant destroy -f" EXIT
 
-curl "${ARCHIVE_URL}" -o submission.zip
-curl "${SCRIPT_URL}" -o checker.sh
+curl "${VMCK_ARCHIVE_URL}" -o submission.zip
+curl "${VMCK_SCRIPT_URL}" -o checker.sh
 
 vagrant up
 vagrant ssh -- < checker.sh > result.out
 
 data="$(base64 result.out)"
 JSON_STRING=$(jq -n \
-                 --arg tok "$SUBMISSION_ID" \
                  --arg out "$data" \
-                 '{token: $tok, output: $out,}')
-curl -X POST "${INTERFACE_ADDRESS}/done/" -d "$JSON_STRING" \
+                 '{output: $out,}')
```

Comment:
Hmm, so the output is basically `stdout`. Let's make this more generic and capture everything we can about that script: stdout, stderr and return code. So the JSON can be `{"stdout": "... base64 of the stdout ...", "stderr": "...", "exit_code": 0}`